frontend/src/components/nav.js

```diff
@@ -39,44 +39,56 @@ class NavigationBar extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      isOpen: false,
+      isLead: false,
```

Comment:
How many levels of user roles are there? If there are more than 2 (or there plan to be), then I'd recommend using an enum to prevent bloat

Reply:
The number of roles doesn't matter here - roles are defined to be 'lead' or 'not lead'. As described in the issue, we should only show the nav components when the user is a lead (which would mean the role could be a lead or a director).